ontology/EBUCorePlus/ebucoreplus.owl

```diff
@@ -3991,6 +4024,11 @@ ec:isTrackPartOf rdf:type owl:ObjectProperty ;
                             "Track part source"@en .
 
 
+###  http://www.ebu.ch/metadata/ontologies/ebucoreplus#isUsedBy
+ec:isUsedBy rdf:type owl:ObjectProperty ;
+            owl:inverseOf ec:uses .
```

Comment:
The two properties are not part of any class restriction, ie. they are not used at all. 
They should both be omitted.